rskj-core/src/test/java/co/rsk/peg/ReleaseTransactionBuilderTest.java

```diff
@@ -516,6 +756,13 @@ private Address mockAddress(int pk) {
         return BtcECKey.fromPrivate(BigInteger.valueOf(pk)).toAddress(NetworkParameters.fromID(NetworkParameters.ID_REGTEST));
     }
 
+    private Address mockAddress(NetworkParameters networkParameters, int keysCount) {
```

Comment:
```suggestion
    private Address createP2SHAddress(NetworkParameters networkParameters, int keysCount) {
```
Maybe that's clearer, also this could well go to PegTestUtils class. It will be useful for other tests

Reply:
Yes, Its Clearer. Done